### PR TITLE
#973 L10n ru: translate lic.jface

### LINKED ISSUE #973: L10n ru: translate lic.jface
We need RU l10n made for `lic.jface`, mainly for all the dialogs participating in licensing status reporting and imroving 
 - licensing status dialog
 - diagnostic dialog
 - license import dialog
 - environment assessment dialog
 - licensing agreements exposure dialog

Prior Babal involvement, simple .properties translaton is to be used.

### PR DESCRIPTION
**Pull request by eparovyshnaya**
l10n for
 - licensing status dialog
 - diagnostic dialog
 - license import dialog
 - environment assessment dialog
 - lic agreements exposure and acceptance dialog

Closes #973

Signed-off-by: eparovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     ArSysOp - initial API and implementation
+###############################################################################
+
+AgreementsWizard_description=\u041F\u0440\u043E\u0447\u0438\u0442\u0430\u0439\u0442\u0435 \u0438 \u043F\u0440\u0438\u043C\u0438\u0442\u0435 \u043B\u0438\u0446\u0435\u043D\u0437\u0438\u043E\u043D\u043D\u044B\u0435 \u0441\u043E\u0433\u043B\u0430\u0448\u0435\u043D\u0438\u044F
+AgreementsWizardDialog_title=\u041E\u0431\u044F\u0437\u0430\u0442\u0435\u043B\u044C\u043D\u044B\u0435 \u043B\u0438\u0446\u0435\u043D\u0437\u0438\u043E\u043D\u043D\u044B\u0435 \u0441\u043E\u0433\u043B\u0430\u0448\u0435\u043D\u0438\u044F
+
+AgreementsWizard_error=\u043F\u0440\u0438\\u043D\u044F\u0442\u0438\u0435 \u043B\u0438\u0446\u0435\u043D\u0437\u0438\u043E\u043D\u043D\u043E\u0433\u043E \u0441\u043E\u0433\u043B\u0430\u0448\u0435\u043D\u0438\u044F \u043D\u0435\u0432\u043E\u0437\u043C\u043E\u0436\u043D\u043E
```
Comment (zelenyhleb) [inline bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties:17]: typo here: при\нятие

File: bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+AgreementsWizard_error_description=\u041A\u043E\u043D\u0444\u0438\u0433\u0443\u0440\u0430\u0446\u0438\u044F \u043F\u0440\u043E\u0434\u0443\u043A\u0442\u0430 \u043D\u0435\u0434\u043E\u0441\u0442\u0430\u0442\u043E\u0447\u043D\u0430, \u043F\u0440\u0438\u043D\u044F\u0442\u0438\u0435 \u043B\u0438\u0446\u0435\u043D\u0437\u0438\u043E\u043D\u043D\u043E\u0433\u043E \u0441\u043E\u0433\u043B\u0430\u0448\u0435\u043D\u0438\u044F \u043D\u0435\u0432\u043E\u0437\u043C\u043E\u0436\u043D\u043E.\
```
Comment (zelenyhleb) [inline bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties:18]: typo: невозможно.\ Свяжитесь с производителем. (\n was meant?)

**Review by zelenyhleb — CHANGES_REQUESTED** [review]
Just two typos.

File: bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties:18]: `\` is a contract for multirow value - finish a row with back slash and the next row is concatenated to it through new-line character.

File: bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/i18n/AgreementsDialogMessages_ru.properties:17]: fixed. Thans for careful reading.

**Review by zelenyhleb — APPROVED** [review]

**Pull request merged**